ui/base-tool/src/main/java/org/cru/godtools/base/tool/activity/ImmersiveActivity.kt

```diff
@@ -1,36 +1,19 @@
 package org.cru.godtools.base.tool.activity
 
-import android.os.Build
 import android.os.Bundle
-import android.view.View
 import androidx.annotation.LayoutRes
-import androidx.annotation.RequiresApi
 import androidx.databinding.DataBindingUtil
 import androidx.databinding.ViewDataBinding
 import org.cru.godtools.base.ui.activity.BaseActivity
 
-abstract class ImmersiveActivity<B : ViewDataBinding>(
-    private val enableImmersive: Boolean,
-    @LayoutRes private val contentLayoutId: Int
-) : BaseActivity() {
+abstract class ImmersiveActivity<B : ViewDataBinding>(@LayoutRes private val contentLayoutId: Int) : BaseActivity() {
```

Comment:
eventually this class will go away, but first the ViewDataBinding management needs to be moved to `BaseActivity`